docs/source/starter/style_guide.rst

```diff
@@ -38,7 +40,7 @@ Here's a LightningModule that defines a model:
 .. testcode::
 
     class LitModel(LightningModule):
-        def __init__(self, num_layers: int = 3):
+        def __init__(self):
```

Comment:
"A LightningModule can define both a system and a model" defeats the point above. 
Users should define pure nn.Modules *outside* of the LightningModule. Including the examples with the layers like this isn't how I've seen the vast majority of users interact with Lightning

Reply:
will update!

Reply:
It's a choice I'd say... although a system is what we recommend but if it's a very simple model, then we don't enforce it.

Reply:
I agree with @ananthsub.

@rohitgr7 How about explicitly mentioning here that users are discouraged to define a "model" like the example (although users can still write that way)?